service/src/main/java/org/cbioportal/service/impl/SampleServiceImpl.java

```diff
@@ -193,6 +198,9 @@ private void processSamples(List<Sample> samples, String projection) {
                 sample.setSequenced(sequencedSampleIdsMap.get(sample.getCancerStudyIdentifier())
                     .contains(sample.getStableId()));
                 sample.setCopyNumberSegmentPresent(samplesWithCopyNumberSegMap.contains(sample.getInternalId()));
+                if (!structuralVariantSampleIdsMap.isEmpty()) {
+                    sample.setFusionPresent(structuralVariantSampleIdsMap.get(sample.getCancerStudyIdentifier()).contains(sample.getStableId()));
```

Comment:
setProfiledForFusions (or whatever rename is chosen)

Reply:
done.